### ISSUE TITLE
You have a problem with apache's "short_open_tag"

### ISSUE THREAD

**Issue by wiwwo** [mongodb/mongo-php-driver#224]
```
user@debBase:/var/www$ lsb_release  -a
No LSB modules are available.
Distributor ID: Debian
Description:    Debian GNU/Linux 8.3 (jessie)
Release:        8.3
Codename:       jessie


user@debBase:/var/www$ uname -a
Linux debBase 3.16.0-4-amd64 #1 SMP Debian 3.16.7-ckt20-1+deb8u3 (2016-01-17) x86_64 GNU/Linux
```

``` 
user@debBase:~# grep short_open_tag /etc/php5/*/php.ini | grep -v ";"
/etc/php5/apache2filter/php.ini:short_open_tag = Off
/etc/php5/apache2/php.ini:short_open_tag = Off
/etc/php5/cli/php.ini:short_open_tag = Off
```

```
user@debBase:/var/www$ cat test.php
<?php
$connection = new Mongo(""mongodb://localhost"");
$db = $connection->database1;
$collection = $db->collection1;
var_dump($collection);
?>
```

(browser) BLANK PAGE

(apache log)
```
[Tue Feb 09 13:08:34.086873 2016] [:error] [pid 1001] [client 192.168.56.1:61709] PHP Fatal error:  Class 'Mongo' not found in /var/www/test.php on line 2
```

```
user@debBase:/var/www$ uname -a
Linux debBase 3.16.0-4-amd64 #1 SMP Debian 3.16.7-ckt20-1+deb8u3 (2016-01-17) x86_64 GNU/Linux



user@debBase:/var/www$ dpkg -l | grep php
ii  dh-php5                               0.2                                all          debhelper add-on to handle PHP PECL extensions
ii  libapache2-mod-php5                   5.6.17+dfsg-0+deb8u1               amd64        server-side, HTML-embedded scripting language (Apache 2 module)
rc  libapache2-mod-php5filter             5.6.17+dfsg-0+deb8u1               amd64        server-side, HTML-embedded scripting language (apache 2 filter module)
ii  php-pear                              5.6.17+dfsg-0+deb8u1               all          PEAR - PHP Extension and Application Repository
ii  php5                                  5.6.17+dfsg-0+deb8u1               all          server-side, HTML-embedded scripting language (metapackage)
ii  php5-cli                              5.6.17+dfsg-0+deb8u1               amd64        command-line interpreter for the php5 scripting language
ii  php5-common                           5.6.17+dfsg-0+deb8u1               amd64        Common files for packages built from the php5 source
ii  php5-curl                             5.6.17+dfsg-0+deb8u1               amd64        CURL module for php5
ii  php5-dev                              5.6.17+dfsg-0+deb8u1               amd64        Files for PHP5 module development
ii  php5-gd                               5.6.17+dfsg-0+deb8u1               amd64        GD module for php5
ii  php5-imagick                          3.2.0~rc1-1                        amd64        Provides a wrapper to the ImageMagick library
ii  php5-imap                             5.6.17+dfsg-0+deb8u1               amd64        IMAP module for php5
ii  php5-intl                             5.6.17+dfsg-0+deb8u1               amd64        internationalisation module for php5
ii  php5-json                             1.3.6-1                            amd64        JSON module for php5
ii  php5-mcrypt                           5.6.17+dfsg-0+deb8u1               amd64        MCrypt module for php5
ii  php5-memcache                         3.0.8-5                            amd64        memcache extension module for PHP5
ii  php5-mysqlnd                          5.6.17+dfsg-0+deb8u1               amd64        MySQL module for php5 (Native Driver)
ii  php5-pspell                           5.6.17+dfsg-0+deb8u1               amd64        pspell module for php5
ii  php5-readline                         5.6.17+dfsg-0+deb8u1               amd64        Readline module for php5
ii  php5-recode                           5.6.17+dfsg-0+deb8u1               amd64        recode module for php5
ii  php5-snmp                             5.6.17+dfsg-0+deb8u1               amd64        SNMP module for php5
ii  php5-sqlite                           5.6.17+dfsg-0+deb8u1               amd64        SQLite module for php5
ii  php5-tidy                             5.6.17+dfsg-0+deb8u1               amd64        tidy module for php5
ii  php5-xmlrpc                           5.6.17+dfsg-0+deb8u1               amd64        XML-RPC module for php5
ii  php5-xsl                              5.6.17+dfsg-0+deb8u1               amd64        XSL module for php5
ii  pkg-php-tools                         1.28                               all          various packaging tools and scripts for PHP packages


user@debBase:/var/www$ dpkg -l | grep apache
ii  apache2                               2.4.10-10+deb8u4                   amd64        Apache HTTP Server
ii  apache2-bin                           2.4.10-10+deb8u4                   amd64        Apache HTTP Server (modules and other binary files)
ii  apache2-data                          2.4.10-10+deb8u4                   all          Apache HTTP Server (common files)
ii  apache2-utils                         2.4.10-10+deb8u4                   amd64        Apache HTTP Server (utility programs for web servers)
rc  libapache2-mod-dnssd                  0.6-3.1                            amd64        Zeroconf support for Apache 2 via avahi
ii  libapache2-mod-php5                   5.6.17+dfsg-0+deb8u1               amd64        server-side, HTML-embedded scripting language (Apache 2 module)
rc  libapache2-mod-php5filter             5.6.17+dfsg-0+deb8u1               amd64        server-side, HTML-embedded scripting language (apache 2 filter module)


root@debBase:~#  pecl info mongodb
About pecl.php.net/mongodb-1.1.2
================================
Release Type          PECL-style PHP extension (source code)
Name                  mongodb
Channel               pecl.php.net
Summary               MongoDB driver for PHP
Description           The purpose of this driver is to provide
                      exceptionally thin glue between MongoDB
                      and PHP, implementing only fundemental and
                      performance-critical components
                      necessary to build a fully-functional MongoDB
                      driver.
Maintainers           Hannes Magnusson <bjori@php.net> (lead)
                      Jeremy Mikola <jmikola@php.net> (lead)
                      Derick Rethans <derick@php.net> (lead)
Release Date          2016-01-07 18:56:56
Release Version       1.1.2 (stable)
API Version           1.1.2 (stable)
License               Apache License
                      (http://www.apache.org/licenses/LICENSE-2.0)
Release Notes         * PHPC-496: Revert "Temporary workaround for
                      warning in mongoc"
                      * PHPC-534: Enable configure without SASL and
                      support --with-mongodb-sasl=no
                      * PHPC-529: Prefix BSON functions to avoid
                      symbol conflicts
                      * PHPC-528: Support zval references when
                      appending BSON
                      * PHPC-527: fix call to add_index_stringl
                      * PHPC-527: Fix PHP 7 / 32bits build
Required Dependencies PHP version 5.4.0-7.99.99
                      PEAR installer version 1.4.8 or newer
package.xml version   2.0
Last Modified         2016-02-09 11:10
Previous Installed    - None -
Version


user@debBase:/var/www$ mongo --version
MongoDB shell version: 3.2.1


user@debBase:/var/www$ mongod --version
db version v3.2.1
git version: a14d55980c2cdc565d4704a7e3ad37e4e535c1b2
OpenSSL version: OpenSSL 1.0.1k 8 Jan 2015
allocator: tcmalloc
modules: none
build environment:
    distmod: debian71
    distarch: x86_64
    target_arch: x86_64

```

**Comment by derickr** [comment]
The following error:

```
[Tue Feb 09 13:08:34.086873 2016] [:error] [pid 1001] [client 192.168.56.1:61709] PHP Fatal error:  Class 'Mongo' not found in /var/www/test.php on line 2
```

is unrelated to short open tags. The class "Mongo" does not exist in the new driver that you are currently have installed. The documentation at http://php.net/manual/en/class.mongoclient.php (which http://php.net/class.mongo points to) mentions that in the new driver, the alternative class is now http://php.net/manual/en/class.mongodb-driver-manager.php

The new "mongodb" driver does not implement the same API as the legacy "mongo" driver. For the history on this, see: https://derickrethans.nl/new-drivers.html and https://derickrethans.nl/new-drivers-part2.html

In other words, there is no bug here.
